doc/sphinx/installation/configuration.rst

```diff
@@ -0,0 +1,99 @@
+.. ---------------------------------------------------------------------
+..
+.. PartExa - A Particle Library for the Exa-Scale
+..
+.. Copyright (C) 2022 by the PartExa authors
+..
+.. This program is free software: you can redistribute it and/or modify
+.. it under the terms of the GNU General Public License as published by
+.. the Free Software Foundation, either version 3 of the License, or
+.. (at your option) any later version.
+..
+.. This program is distributed in the hope that it will be useful,
+.. but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+.. GNU General Public License for more details.
+..
+.. You should have received a copy of the GNU General Public License
+.. along with this program. If not, see <https://www.gnu.org/licenses/>.
+..
+.. ---------------------------------------------------------------------
+
+.. toctree::
+   :hidden:
+
+.. _configuration:
+
+Configuration and installation
+------------------------------
+
+For configuration and installation of PartExa and the associated Sphinx and Doxygen documentation
+first clone the PartExa source code repository:
+
+.. code-block:: console
+
+   git clone git@github.com:PartExa/PartExa.git <partexa_source>
+
+PartExa library
+~~~~~~~~~~~~~~~
+
+In the following it is assumed that all :ref:`dependencies` are met. Configure and build PartExa
+following
+
+.. code-block:: console
+
+   cd <partexa_build>
```

Comment:
Include `mkdir <partexa_build>` for completeness?

Reply:
I'd like to keep the guide brief and would rather add one note above that states that all directory paths embraced in `<...>` need to be created by the user. What do you think?

Reply:
Imo adding the mkdir directives wouldn't add too much code and makes it easier to follow the steps.

Reply:
Added all `mkdir` as discussed yesterday.